modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4336,6 +4343,20 @@ else if (StringUtil.equals(
 		}
 	}
 
+	private void _validateWorkflowAction(
+			Integer currentStatus, ObjectDefinition objectDefinition,
```

Comment:
What do you think to change `currentStatus` to `objectEntryStatus`? Sort with this new parameter name.

Reply:
I believe that we can pass only `boolean enableObjectEntryDraft` to this method, what do you think?